docker-compose.yml

```diff
@@ -46,8 +46,11 @@ services:
         build:
           context: ./osrm
           dockerfile: Dockerfile
+        image: osrm
         volumes:
           - ./data/:/data/:rw
+          - ./osrm:/osrm/:ro
+        command: sh -c "python3 /osrm/initialize.py"
```

Comment:
You could also make the file executable, so that the python version is reference in the initialize.py itself

Reply:
Done.